server/webapp/WEB-INF/rails.new/spec/views/admin/templates/index_html_spec.rb

```diff
@@ -283,16 +351,38 @@
     end
   end
 
-  it "should disable the delete button next to the template name for template admin" do
+  it "should display the delete button next to the template name for template admin" do
```

Comment:
https://github.com/gocd/gocd/issues/3081#issuecomment-274057578 - We discussed and we didn't know why template admins were not allowed to do this before. It made sense before because only those who can create a template should be able to delete it. Then, we discussed that pipeline group admins can create a template and be made template admins by default. So, if pipeline group admins can create a template, then they should be able to delete it as template admins, which means all template admins should be able to delete it.